functionary/train/train_lora.py

```diff
@@ -47,7 +42,11 @@ class ModelArguments:
 @dataclass
 class DataArguments:
     train_data_path: str = field(default=None, metadata={"help": "Path to the training data."})
+    training_ratio: float = field(default=1.0, metadata={"help": "percentage of data used for training"})
     eval_data_path: str = field(default=None, metadata={"help": "Path to the eval data."})
+    eval_ratio: float = field(default=1.0, metadata={"help": "percentage of data used for evluation"})
+    packing: bool = field(default=False, metadata={"help": "Whether use packing or not"})
+    max_packing_length: int = field(default=4096, metadata={"help": "maximum sequence length we can pack"})
```

Comment:
shouldn't this be same as the max context length?

Reply:
Ah I forgot to remove it